app/assets/stylesheets/main.scss

```diff
@@ -493,4 +493,424 @@ a,
   list-style-type: disc;
 }
 
-//classic stuff end
\ No newline at end of file
+//classic stuff end
+
```

Comment:
Not sure what all this stuff is but would guess it's a shim for updating to the new accessiblity stuff.

Should it be in its own SCSS file that's given a name that lets us know what its for? That way it'd be kept separate from this service's specific style stuff that we maintain for this service.